src/main/com/example010/radek/v1/IBaseFighter.java

```diff
@@ -0,0 +1,10 @@
+package main.com.example010.radek.v1;
+
+public interface IBaseFighter {
+
+    public int shooting();

```

Comment:
vse, co je v interface je public, takze vsude smazat

Reply:
Ok, už jsem si všiml, že to bylo šedé, dobré vědět